pixeltable/index/embedding_index.py

```diff
@@ -127,7 +130,13 @@ def records_value_errors(self) -> bool:
 
     def get_index_sa_type(self, val_col_type: ts.ColumnType) -> sql.types.TypeEngine:
         assert isinstance(val_col_type, ts.ArrayType) and val_col_type.shape is not None
+        assert len(val_col_type.shape) == 1, (
+            f'Pixeltable does not support multi-dimensional embedding vectors: {val_col_type}'
```

Comment:
I'd say "Expected a 1-dimensional array, got {n}: ..."

Technically a vector is 1-dimensional so "multi-dimensional vector" is not a meaningful term. Multi-vector embeddings are a separate concept so here it's not really a question of what's supported or not, it's a question of what inputs are meaningful.